spec/features/posts_show_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.feature 'User Index Page', type: :feature do
+  before do
+    @user1 = User.create(name: 'Sam', photo: 'https://icons.iconarchive.com/icons/iconsmind/outline/256/User-icon.png',
+                         bio: 'Teacher from Poland.', post_counter: 0)
+    visit users_path
+  end
+
+  scenario 'I can see the username, profile picture, and number of posts for each user' do
+    expect(page).to have_content(@user1.name)
+    expect(page.html).to include(@user1.photo)
+    expect(page).to have_content(@user1.post_counter)
+  end
```

Comment:
You are suppose to write at least 7 tests but I can see only 3 so far. Kindly ensure that all tests cases are covered as indicated in the project requirements. :pray: 

![Screenshot from 2023-06-21 11-53-28](https://github.com/edubew/blog-box/assets/65233860/1493f698-2dd9-431f-b894-c804dc7c7ccd)